scripts/gen_dictionaries.py

```diff
@@ -127,7 +127,9 @@ def GetFooter():
     for enums in spec.findall('enums'):
         # Skip Vendor Extension Enums
         vendor = enums.get('vendor')
-        if not vendor or vendor == 'Khronos' or vendor == 'Multiple':
+        name = enums.get('name')    # special-case: enum block with KHR enums assigned to vendor
+        include_anyway = name == 'enums.4010'
```

Comment:
No issue with this, but does this just happen when an extension goes from being a Vendor extension to a KHR?

Reply:
I filed https://github.com/KhronosGroup/OpenCL-Docs/issues/431 so we don't lose track of this.  I agree this currently solution is a bit of a hack.

I'll also see if the script could also just generate dictionaries for all enums, if there is no downside to doing this.